.github/workflows/gradle_publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Gradle Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Create local.properties with a read token
+      run: |
+        # Use a personal read token to install the Cyface Utils package
+        echo "github.user=${{ secrets.GITHUB_READ_ACCOUNT }}" > local.properties
+        echo "github.token=${{ secrets.GITHUB_READ_TOKEN }}" >> local.properties
+    - name: Build with Gradle
+      run: ./gradlew build
+    - name: Create local.properties with a repository token
+      run: |
+        # Use the repository's automatically set up token to pushlish to the registry
+        echo "github.user=cyface-de" > local.properties
+        echo "github.token=${{ secrets.GITHUB_TOKEN }}" >> local.properties
```

Comment:
To *publish* this package automatically (as introduced in this commit) Github automatically creates a temporary access token with write access (only to this repository)  when the Github Actions are triggered. This token is used here.